src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -82,7 +82,7 @@ namespace NuGetVSExtension
     public sealed class NuGetPackage : AsyncPackage, IVsPackageExtensionProvider, IVsPersistSolutionOpts
     {
         // It is displayed in the Help - About box of Visual Studio
-        public const string ProductVersion = "6.1.0";
+        public const string ProductVersion = "6.2.0";
```

Comment:
If I added build logic to generate this at build time from `build\config.props`, would that be useful to us?  Or do we have a need to have this version differ in some cases?

Reply:
it would be useful, it's just that this is so trivial to update that it's not worth spending an hour to fix because that's the same effort that updating this for 60 versions would take. But I'll wholly support anyone who creates a PR for it.

Personally, I think it would be nice if it used the actual product version, rather than hardcoding the last digit to zero. That would make it a little bit easier for us when customers report version based on what's displayed in VS's About window.

Reply:
Please take a look at https://github.com/NuGet/NuGet.Client/pull/4395 to see if it does what you wanted

Reply:
I like this idea!

Andy overlooked the cost of a PR, running CI, the overall distraction that even 10 minutes poses on hotseat. 
One less thing to worry about sounds great.

Reply:
A PR, and its build, will still be needed to update `config.props`, so I don't understand the comment 🤷

Reply:
Oh right, even if we update props, we still build VS entirely. So not helping our efficiency at all, is it?